style.css

```diff
@@ -0,0 +1,85 @@
+.ul {
+  list-style: none;
+}
+
+.main-section {
+  background-image: url('img/bg.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  min-height: 800px;
+}
+
+.text-section {
+  margin: 0 260px 0 260px;
```

Comment:
- Hey, please, kindly reduce this margin-right and margin-left, as its making your design look very different. kindly make this adjustment to improve your design.